### DISCUSSION TITLE
operation descriptions from operation index, ._. etc.

### DISCUSSION THREAD

**Discussion by tattwamasi** [commonism/aiopenapi3#194] [Q&A] [answered]
Hi,
I deeply appreciate this repo.  Found it from the original you forked from and discussions there.  It's been really handy, but I have a question about the description field and such from the openapi doc.  I sort of expected something like myapi._.myfunc.description to work.  It doesn't, but it looks like the info is in the model object in paths.  
Am I missing something obvious about the expected way one would quickly get an operation description if one has the OperationID (for example, from the iterator)?

To be clear, I'm asking specifically about the operations themselves. I can get the parameters and their descriptions, etc. just fine.

It's worked around (by doing a pass through the pathitems and grabbing descriptions that way) but just curious if I missed something.

Sorry if it's a silly question, and thanks again for the work!

**Comment by commonism** [comment, marked as answer]
Hi,

```python
myapi._.myfunc
```
the sad smiley interface - is basically syntactic sugar for `api.createRequest()`.
RequestBase has .operation

https://github.com/commonism/aiopenapi3/blob/e9294f24ad5d7fa219625d401cbd0ad323893f45/aiopenapi3/request.py#L94

access the Operation via `myapi._.myfunc.operation`.

Currently missing as [documentation](https://aiopenapi3.readthedocs.io/en/latest/api.html#aiopenapi3.request.RequestBase), will take care.

**Reply by commonism** [reply]
I guess it's [unlikely](https://aiopenapi3.readthedocs.io/en/latest/api.html#aiopenapi3.request.RequestBase.operation) to be found, but I tried and the [other](https://github.com/commonism/aiopenapi3/commit/0d88a26072e60996089e356820aca0869c5624be) changes may be more useful.
